README.md

```diff
@@ -39,13 +39,33 @@ All messages                         | Alpha    |
 (See [maturity-matrix.yaml](https://github.com/open-telemetry/community/blob/47813530864b9fe5a5146f466a58bd2bb94edc72/maturity-matrix.yaml#L57)
 for definition of maturity levels).
 
-Note that maturity guarantees apply only to wire-level compatibility for the binary
-Protobuf serialization. Neither message, field, nor enum names of Protobuf messages
-are visible on the wire and are not considered part of the guarantees. We are free
-to make a change to the names.
+## Stability Definition
 
-In the future when OTLP/JSON is declared stable, field names will also become part of
-the maturity guarantees, since field names are visible on the wire for JSON encoding.
+Components marked `Stable` provide the following guarantees:
+
+- Field types will not change.
+- Field numbers will not change.
+- Numbers assigned to enum choices will not change.
+- Service names and service package names will not change.
+- Service operation names, parameter and return types will not change.
+
+The following changes are allowed:
+
+- Message names may change.
+- Field names may change.
+- Enum names may change.
+- Enum choice names may change.
```

Comment:
Is it possible to add more Enum values? Above it just says `- Numbers assigned to enum choices will not change.`, which would still be fulfilled if adding more choices, right?

Reply:
Sounds good, do we need to call it out in the list of allowed changes? This is also somewhat related to @reyang's [comment](https://github.com/open-telemetry/opentelemetry-proto/pull/406#discussion_r908634739), as it would be UNDEFINED if we don't add it.